MIP0/mip0.md

```diff
@@ -7,287 +11,254 @@ Title: The Maker Improvement Proposal Framework
 
-**MIP0c2: Core Principles**  
-Discusses some core principles that all MIPs should aim to follow.
 
-**MIP0c3: The MIP Lifecycle**  
-Lays out how a MIP is created and moves through the process to become Accepted or Rejected.
 
-**MIP0c4: MIP Components and MIP Component Types**  
-Discusses the use of components to compartmentalize and organise MIPs
+## Motivation
 
-**MIP0c5: MIP Replacement Process**  
-Discusses how MIPs can be replaced and the steps to be taken to maintain dependencies.
+MakerDAO is evolving into a trustless, fully decentralized, open-sourced, and self-sustainable organization. By offering a standardized, open-to-everyone approach to the proposal of changes to the protocol and governance, Maker Improvement Proposals (MIPs) make it possible for the Maker community to exercise long-term decentralized self-governance.
 
-**MIP0c6: Supporting Materials**
-A component that defines how to include external materials inside MIPs.
 
-**MIP0c7: MIP Templates**  
-Defines the MIP templates for both General and Technical MIPs.
+## Specification / Proposal Details
 
-**MIP0c8: MIP0 Domain Role Dependencies**  
-Defines the core roles that the MIPs process requires to operate successfully.
+### MIP0c1: Definitions of the Maker Improvement Proposal Framework
 
-**MIP0c9: Core Personnel Role List**  
-List of of personnel currently occupying core roles.
+- **Maker Improvement Proposals (MIPs):** - **Maker Improvement Proposals (MIPs):** MIPs are standardized documents subject to voting that (once enacted) regulate and define the behavior of the Maker Governance and the Maker Protocol. MIPs can be either general or technical (see `MIP0c4`). MIPs can be added, amended, replaced, and removed.
+-   **MIP Sets:** MIP Sets are groups of interdependent MIPs that work as a modular whole. See `MIP0c12`.
+-   **MIP Types:** MIPs can be either general, technical, or process MIPs. See `MIP0c5`.
+-   **Subproposals (SPs):** Subproposals are instances of Process MIP Components. See `MIP0c5`.  
+-   **Minimum Feedback Period:** The minimum amount of time within which the community is able to give feedback in response to a proposed MIP.  
+-   **Minimum Frozen Period:** The minimum amount of time during which a MIP must remain unchanged (frozen) before it can be submitted for ratification/implementation.
+-   **Governance Facilitator(s):** Governance Facilitators are tasked with ensuring a smooth operation of the governance process. This involves a wide range of activities, anything from general administration through signals gathering to governance scheduling. Governance Facilitators cannot delegate their responsibilities in the MIP Framework and process that require their formal involvement.
+-  **MIP Editor(s):** MIP Editors enforce the administrative and editorial aspects of the overall MIPs process and program. Governance Facilitators are granted the MIP Editor role, but can also delegate the MIP Editor role to contributors.
```

Comment:
Facilitators versus MIP Editors seems contradictory? Facilitators cannot delegate, but can delegate to contributors?